aptos-move/aptos-sdk-builder/src/common.rs

```diff
@@ -69,7 +69,7 @@ pub(crate) fn make_abi_enum_container(abis: &[ScriptABI]) -> ContainerFormat {
         }
 
         let name = match abi {
-            ScriptABI::ScriptFunction(sf) => {
+            EntryABI::EntryFunction(sf) => {
```

Comment:
This is what will fail unless you updated Move.

Reply:
I made my hack by copying the type ... we'll need to update move and delete that type :)